tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -94,363 +121,426 @@ Future<void> performCreate(
   var awsName = name.replaceAll('_', '-');
   var randomAwsId = math.Random.secure().nextInt(10000000).toString();
 
-  var projectDir = Directory(p.join(Directory.current.path, name));
-  if (projectDir.existsSync()) {
+  var serverpodDirs = ServerpodDirectories(
+    projectDir: Directory(p.join(Directory.current.path, name)),
+    name: name,
+  );
+  if (serverpodDirs.projectDir.existsSync()) {
     log.error('Project $name already exists.');
-    return;
+    return false;
   }
 
-  log.info(
-    'Creating project $name.',
-    style: const TextLogStyle(newParagraph: true),
-  );
+  if (template == ServerpodTemplateType.server) {
+    log.info(
+      '🚀 Creating Serverpod project "$name".',
```

Comment:
This will create an empty space at the start of the line on Windows? How about we place the rocket at the end of the line?